include/swift/SIL/SILCloner.h

```diff
@@ -181,32 +157,46 @@ class SILCloner : protected SILVisitor<ImplClass> {
         return t;
       })->getCanonicalType();
   }
+
   CanType getOpASTType(CanType ty) {
     ty = getASTTypeInClonedContext(ty);
     return asImpl().remapASTType(ty);
   }
 
-  /// Remap an entire set of conformances.
-  ///
-  /// Returns the passed-in conformances array if none of the elements
-  /// changed.
-  ArrayRef<ProtocolConformanceRef> getOpConformances(CanType type,
-                             ArrayRef<ProtocolConformanceRef> oldConformances) {
-    Substitution sub(type, oldConformances);
-    Substitution mappedSub = asImpl().remapSubstitution(sub);
-    ArrayRef<ProtocolConformanceRef> newConformances =
-      mappedSub.getConformances();
-
-    // Use the existing conformances array if possible.
-    if (oldConformances == newConformances)
-      return oldConformances;
-
-    return type->getASTContext().AllocateCopy(newConformances);
+  ProtocolConformanceRef getOpConformance(Type ty,
+                                          ProtocolConformanceRef conformance) {
+    auto newConformance =
+      conformance.subst(ty,
+                        [&](SubstitutableType *t) -> Type {
+                          if (t->isOpenedExistential()) {
```

Comment:
Use llvm's early return style here. `if (!t-> isOpenedExistential()) return t;` This will reduce the indentation.